G/gdk_pixbuf/build_tarballs.jl

```diff
@@ -49,19 +66,23 @@ products = [
 # Some dependencies are needed only on Linux and FreeBSD
 linux_freebsd = filter(p->Sys.islinux(p)||Sys.isfreebsd(p), platforms)
 
+# gobject_introspection is needed only on x86_64-linux-gnu
+introspect_platform = filter(p -> triplet(p) == "x86_64-linux-gnu", platforms)
```

Comment:
I prefer more declarative checks, checking the triplet is problematic in the case in which we need to expand the list of platforms above with more tags
```suggestion
introspect_platform = filter(p -> Sys.islinux(p) && libc(p) == "glibc" && arch(p) == "x86_64", platforms)
```